wormhole-connect/src/AppRouter.tsx

```diff
@@ -80,9 +79,9 @@ function AppRouter(props: Props) {
       config.whLegacy.registerProviders(); // reset providers that may have been set during transfer
     }
     // reset transfer state on leave
-    if (route === bridgeRoute && prevRoute !== bridgeRoute) {
+    const isEnteringBridge = route === bridgeRoute && prevRoute !== bridgeRoute;
+    if (isEnteringBridge && prevRoute !== 'history') {
       dispatch(clearTransfer());
-      dispatch(clearPorticoBridge());
```

Comment:
We should remove this from the store, but it has cascading changes, so didn't want to do it here

Reply:
can you add `TODO SDKV2` comment somewhere to remind us?

Reply:
done